Repository/0.5.2.0/Gess1t/Gess1tsProxyAPI/ProxyAPI/1.0.0.json

```
@@ -0,0 +1,13 @@
+{
+    "Name": "Gess1t's Proxy API",
+    "Owner": "Gess1t",
+    "Description": "Interact will plugins, Make Overlays and more.",
```

Grammar is quite odd here, could probably be changed up to something like this
```suggestion
    "Description": "Proxy library for external utilities",
```